sympy/logic/boolalg.py

```diff
@@ -2769,20 +2769,24 @@ def simplify_logic(expr, form=None, deep=True, force=False):
         made. By setting ``force`` to ``True``, this limit is removed. Be
         aware that this can lead to very long simplification times.
 
+    dontcare : Boolean expression
```

Comment:
Speaking as someone who only recently learned what "don't care" means in logic maybe it would be good to have a reference for this:
https://en.wikipedia.org/wiki/Don%27t-care_term